c2corg_ui/static/js/imageuploader.js

```diff
@@ -227,7 +233,8 @@ app.ImageUploaderController.prototype.uploadFile_ = function(file) {
     'metadata': {
       'id': file['name'] + '-' + new Date().toISOString(),
       'activities': [],
-      'categories': []
+      'categories': [],
+      'image_type': this.image_type_
```

Comment:
Do you really need this intermediate property ``this.image_type_``?
Why not simply:
```
'image_type': this.getImageType_()
```